### ISSUE TITLE
Feature request: airlines

### ISSUE THREAD

**Issue by olejorga** [signicode/openflights-cached#2]
Any plans of adding the airline data from Openflights as well? :)

**Comment by MichalCz** [comment]
I think I could have such... How would you see it? Any interface suggestions?

**Comment by olejorga** [comment]
Could be done similar to with airports. All airlines have icao and iata codes so it would be possible, I think. But I guess you would have to separate airlines from airports in some way. 
E.g. openflights.**airlines**.findIATA("PEK").name

**Comment by MichalCz** [comment]
Yes, makes sense... Ok - I'm quite busy with some other work currently, but I can try at some point.

On the other hand - it's a matter of rewriting `bin/fetch.js` a bit in `bin/fetch-airlines.js` and then creating some getters for this. If you'd like to create a PR I'd try to help out and take it. :)

**Comment by olejorga** [comment]
Ok, I'll give it a shot
